yarn/datadog_checks/yarn/yarn.py

```diff
@@ -144,6 +147,16 @@ class YarnCheck(AgentCheck):
 
     _ALLOWED_APPLICATION_TAGS = ['applicationTags', 'applicationType', 'name', 'queue', 'user']
 
+    def __init__(self, *args, **kwargs):
+        super(YarnCheck, self).__init__(*args, **kwargs)
+        application_status_mapping = self.instances[0].get('application_status_mapping', {})
```

Comment:
Would be great to provide a default mapping. Like the one in the example above `conf.yaml.example`. Otherwise, it will be always `UNKNOWN` if the user doesn't define one.